### DISCUSSION TITLE
Interface with League-Page

### DISCUSSION THREAD

**Discussion by Robert-litts** [joeyagreco/leeger#22] [Ideas]
Hey, I commented on your reddit post and absolutely love this! I have been working with [espn-api](https://github.com/cwendt94/espn-api/wiki) since last year and my limited programming skills have made it quite challenging. When I saw this, I was blown away at how simple it was to use. I tested it out last night and was able to get 9 years of data onto my [League-Page](https://demboys.vercel.app/) using your code. I'm not sure if you have seen [Nicholas Melhado's League-Page](https://github.com/nmelhado/league-page) before, but I think it would be the perfect place to utilize the code you have here. There is clearly a lack of support for Sleeper with historical ESPN/Yahoo Fantasy League data, and the website template he created would be an awesome landing spot to allow people moving over from these other platforms to Sleeper to keep their league history alive. I just made a post in his github page on an issue that someone had put in requesting historical league data from other platforms, so if you have a chance to [check it out ](https://github.com/nmelhado/league-page/issues/66#issuecomment-1237086039)that would be awesome. I really think that this work combined with League-Page would provide the perfect transition piece for those ESPN/Yahoo hold-overs who want to make the switch to Sleeper but really don't want to lose their past records. When I showed my league the stats you provided, they thought it was awesome and honestly its 1000% better than the terrible interface that ESPN has for past records. Let me know what you think & thanks again for sharing this work, truly impressive.

**Comment by joeyagreco** [comment]
Hey Robert,

Thanks for checking out this repo! 

I had not heard of League Page before this but I just looked over it and it's very impressive!

I think adding a stats section with each year/all-time would be very fitting there and a great enhancement to the website.

There are some features I'm currently working on bringing to leeger right now but once those are finished I will definitely look at implementing a way to see leeger stats in League Page (assuming that its wanted).

**Reply by Robert-litts** [reply]
Joey,

Awesome to hear. From reading some past discussions on League-Page, looks like Nicholas was looking for some help trying to get the ESPN/Yahoo data into his site, so this could be an excellent opportunity. I'll see what he says back. If he's amenable to it, I would love to assist wherever I can.
